src/js/pages/Index.js

```diff
@@ -190,6 +195,19 @@ var Index = React.createClass({
     }
   },
 
+  getLanguage() {
```

Comment:
it does a bit more than just getting a Language :) How do you feel about wrapping language related manipulations `UserSettingsStore.getKey(SAVED_STATE_KEY)` with a class?  Like `UserLanguageStore.set` `UserLanguageStore.get`?